packages/tradershub/src/screens/CurrencySelector/CurrencyCard.tsx

```diff
@@ -21,11 +21,11 @@ const CurrencyCard = ({ icon: Icon, id, info, title }: TCurrencyCard) => {
     const { setFieldValue, values } = useFormikContext<{ currency: string }>();
     const isSelected = values.currency === id;
     return (
-        <div className='relative flex justify-center w-1/4 my-400'>
-            <div
+        <div className='relative flex justify-center w-1/2 md:w-1/4 my-400'>
+            <button
                 className={qtMerge(
                     `w-10/12 rounded-400 py-1100 hover:cursor-pointer hover:outline outline-1 ${
-                        isSelected ? 'outline outline-2' : ''
+                        isSelected ? 'outline outline-2 outline-brand-blue' : ''
```

Comment:
```
qtMerge('...base class...', {
   'outline-2 outline-brand-blue': isSelected
})
```
I don't even think you need `outline` because you already have `outline-2` (correct me if I'm wrong)

Reply:
`outline-2` is the `outline-width` while `outline` defines the `outline-style: solid`

Reply:
```
qtMerge('...base class...', {
   'outline-2 outline-brand-blue': isSelected
})```

qtMerge doesn't have this API like you suggested